.github/workflows/triggers-dev.yml

```diff
@@ -5,9 +5,23 @@ on:
       - main
 
 jobs:
+  # ensure that every time a push to main occurs, the container is built and published to ghcr
+  build-container:
+    uses: ./.github/workflows/build-docker-container.yml
+    secrets: inherit
+    permissions:
+      contents: read
+      packages: write
+    with:
+      docker-name: fac
+      image-name: web-container
+      repo-name: gsa-tts/fac
+      work-dir: ./backend
+
   # testing
   test-and-lint:
-    uses: ./.github/workflows/test.yml
+    needs: [build-container]
+    uses: ./.github/workflows/test-and-lint-from-ghcr.yml
```

Comment:
**question (non-blocking)**: Do we know at this point that the new GHCR image from the PR to main has been built and pushed to GHCR and will be the one pulled by this test run? It'd be a shame to fail a test run at this point because the new dependencies weren't quite there in GHCR yet.

Reply:
`triggers-dev.yml` has a [push hook on main](https://github.com/GSA-TTS/FAC/blob/64ea9e3ef2858d53add14ac116db025dbfafb75d/.github/workflows/triggers-dev.yml#L2-L5C13). So every time we merge, before it tests and lints (regardless of whether there were changes to the 4 files or not) we will [build the image and publish to ghcr](https://github.com/GSA-TTS/FAC/blob/64ea9e3ef2858d53add14ac116db025dbfafb75d/.github/workflows/triggers-dev.yml#L9-L19) by invocation of [build-docker-container.yml](https://github.com/GSA-TTS/FAC/blob/64ea9e3ef2858d53add14ac116db025dbfafb75d/.github/workflows/build-docker-container.yml)

So on merge to `main` we will publish a new `web-container:latest` and then test with the ghcr image.

At 6a EST M-F, as part of the nightly deploy to staging, it will trivy scan, build and publish, test with gchr image, and create the PR for review [staging-scheduled-deploy.yml](https://github.com/GSA-TTS/FAC/blob/64ea9e3ef2858d53add14ac116db025dbfafb75d/.github/workflows/staging-scheduled-deploy.yml). More info/diagram on #1334